src/Tooltip/Tooltip.js

```diff
@@ -4,19 +4,36 @@ import styled from 'styled-components';
 
 import { shadow } from '../common';
 
+const DownStyle = `
+  bottom: -4px;
+  transform: translate(-50%, 100%);
+`;
+const UpStyle = `
+  top: -4px;
+  transform: translate(-50%, -100%);
+`;
+const LeftStyle = `
+  transform: translate(-150%, 0);
+`;
+const RightStyle = `
+  transform: translate(50%, 0);
+`;
+
```

Comment:
There are some issues with `left` and `right` positions. See the screenshot below:

![Screenshot 2022-01-23 at 14 23 35](https://user-images.githubusercontent.com/28541613/150680603-220a25d8-1037-4fc4-9d7c-c40bc36bb126.png)

To reproduce the issue just make the hoverable element bigger (like on the screenshot)